analysis/regionEventYields.cpp

```diff
@@ -626,64 +649,106 @@ MT2Analysis<T>* computeSigYield( const MT2Sample& sample, const MT2Config& cfg )
     int GenSusyMScan1=0;
     int GenSusyMScan2=0;
     if(  myTree.evt_id > 999){
+     
+      if(sigSampleName.Contains("T2qq")){
+	
+	GenSusyMScan1 = myTree.GenSusyMSquark;
+	GenSusyMScan2 = myTree.GenSusyMNeutralino;
+
+      }
+      else if(sigSampleName.Contains("T2bb")){
+	
+	GenSusyMScan1 = myTree.GenSusyMSbottom;
+	GenSusyMScan2 = myTree.GenSusyMNeutralino;
+	
+      }
+      else if(sigSampleName.Contains("T2tt")){
+	
+	GenSusyMScan1 = myTree.GenSusyMStop;
+	GenSusyMScan2 = myTree.GenSusyMNeutralino;
+
+      }
+      else{
+      
       GenSusyMScan1 = myTree.GenSusyMGluino;
       GenSusyMScan2 = myTree.GenSusyMNeutralino;
+      
+      }
+
     }
  
     //Double_t weight = (myTree.isData) ? 1. : myTree.evt_scale1fb*cfg.lumi()*myTree.puWeight;
-    Double_t weight = (myTree.isData) ? 1. : myTree.evt_scale1fb;//*cfg.lumi();
-    // Double_t weight = (myTree.isData) ? 1. : myTree.evt_scale1fb*cfg.lumi();
+    //    Double_t weight = (myTree.isData) ? 1. : myTree.evt_scale1fb;//*cfg.lumi();
+    Double_t weight = (myTree.isData) ? 1. : myTree.evt_scale1fb*cfg.lumi(); //Keeping normalization to luminosity for signal?
```

Comment:
I'd prefer to not scale the weight by lumi as we also don't do it anymore for the background MC